gm4_trash_cans/data/gm4_trash_cans/functions/check_rate.mcfunction

```diff
@@ -0,0 +1,13 @@
+# @s = trash_can that has items in it and has not been disabled
+# run from check_dropper
+
+# Set the rate of deletion
+execute if block ~ ~ ~ dropper[triggered=false] if entity @a[distance=..15,gamemode=!spectator] run scoreboard players set @s gm4_trash_rate 0
+execute if block ~ ~ ~ dropper[triggered=false] unless entity @a[distance=..15,gamemode=!spectator] run scoreboard players set @s gm4_trash_rate 16

```

Comment:
```suggestion
execute if block ~ ~ ~ dropper[triggered=false] unless entity @p[distance=..15,gamemode=!spectator] run scoreboard players set @s gm4_trash_rate 16

```

Reply:
Is this better than `@a[distance=..15,gamemode=!spectator,limit=1]`?